Explorer/Assets/DCL/AvatarRendering/Wearables/Components/IWearable.cs

```diff
@@ -21,14 +23,73 @@ public interface IWearable : IAvatarAttachment
         /// </summary>
         bool TryResolveDTO(StreamableLoadingResult<WearableDTO> result);
 
-        void ResolvedFailedDTO(StreamableLoadingResult<WearableDTO> result);
-
         bool TryGetFileHashConditional(BodyShape bodyShape, Func<string, bool> contentMatch, out string? hash);
 
         void GetHidingList(string bodyShapeType, HashSet<string> hideListResult);
 
         bool IsCompatibleWithBodyShape(string bodyShape);
 
         bool HasSameModelsForAllGenders();
+
+        class Fake : IWearable
```

Comment:
Maybe we can move this cumbersome fake to the testing domain?

Reply:
Sure